testing/src/main/java/org/oppia/android/testing/KonfettiViewMatcher.kt

```diff
@@ -11,25 +11,36 @@ class KonfettiViewMatcher {
     /**
      * This function checks that the view being checked is a [KonfettiView] with an active confetti animation.
      *
-     * @returns a [TypeSafeMatcher] that performs matching on the view
+     * @returns a [TypeSafeMatcher] that checks if the given view is a [KonfettiView] that isActive with
+     *          confetti
      */
     fun hasActiveConfetti(): TypeSafeMatcher<View> {
       return ActiveConfettiMatcher()
     }
 
     /**
-     * A custom class to check that the associated view is a [KonfettiView] and isActive().
-     *
-     * @returns a [TypeSafeMatcher] that checks if the given view is a [KonfettiView] that isActive with
-     *          confetti
+     * A custom [TypeSafeMatcher] class to check that the associated view is a [KonfettiView] and isActive().
      */
     private class ActiveConfettiMatcher() : TypeSafeMatcher<View>() {
+      var view: KonfettiView? = null
+
       override fun describeTo(description: Description) {
-        description.appendText("KonfettiView with active confetti animation")
+        if (view != null) {
+          description.appendText(
+            String.format(
+              "KonfettiView with %s active confetti systems", view!!.getActiveSystems()
```

Comment:
You can use Kotlin string interpolation, instead: ``"KonfettiView with ${view?.activeSystems} active confetti systems"``. That being said, it seems like this is a debug change so please revert.

Reply:
Done